docs/Running Migrations.md

```diff
@@ -63,11 +64,17 @@ defmodule MyApp.ReleaseTasks do
     :init.stop()
   end
 
+  def migrate, do: Enum.each(repos(), &run_migrations_for/1)
+
   def priv_dir(app), do: "#{:code.priv_dir(app)}"
 
-  defp run_migrations_for(app) do
-    IO.puts "Running migrations for #{app}"
-    Ecto.Migrator.run(MyApp.Repo, migrations_path(app), :up, all: true)
+  defp run_migrations_for(repo) do
+    repo.config
```

Comment:
I'm not a huge fan of forcing things in to pipes, can you modify this to be:

```elixir
app = Keyword.get(repo.config, :otp_app)
IO.puts "Running migrations for #{app}"
Ecto.Migrator.run(repo, migrations_path(app), :up, all: true)
```

Reply:
applied